db/migrate/20210204144205_create_follows.rb

```diff
@@ -0,0 +1,10 @@
+class CreateFollows < ActiveRecord::Migration[6.1]
```

Comment:
@idprietop is this related to the login task?

Reply:
It is not related to that task, surely I added it before uploading the PR and forgot to remove it.

Reply:
ok! Thanks!